extension/indexer/tx_indexer.go

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package indexer
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/consts"
+	"github.com/ava-labs/hypersdk/fees"
+)
+
+var (
+	_ AcceptedSubscriber = (*txIndexer)(nil)
+	_ AcceptedSubscriber = (*noopTxIndexer)(nil)
+)
+
+var (
```

Comment:
nit: combine the two `var` blocks

Reply:
I generally prefer a separate `var` block for interface assertions, but don't see that used elsewhere in this repo and do see a combined `var` block with another variable used in one place.

Will update.

Reply:
Done